apps/main/src/dex/components/PagePool/index.tsx

```diff
@@ -229,68 +231,74 @@ const Transfer = (pageTransferProps: PageTransferProps) => {
 
       <Wrapper isAdvanceMode={true} chartExpanded={chartExpanded}>
         <AppPageFormsWrapper className="grid-transfer">
-          <Stack sx={{ backgroundColor: (t) => t.design.Layer[1].Fill }}>
+          <Stack
+            sx={{
+              width: { mobile: '100%', tablet: MaxWidth.dexActionCard, desktop: MaxWidth.dexActionCard },
+              marginLeft: { mobile: 'auto', tablet: 'auto', desktop: 0 },
+              marginRight: { mobile: 'auto', tablet: 'auto', desktop: 0 },
```

Comment:
The mui breakpoints apply to anything larger

```suggestion
              width: { mobile: '100%', tablet: MaxWidth.dexActionCard },
              marginLeft: { mobile: 'auto', desktop: 0 },
              marginRight: { mobile: 'auto', desktop: 0 },
```